NineChronicles.Headless.Executable/Program.cs

```diff
@@ -190,12 +195,22 @@ static void ConfigureSentryOptions(SentryOptions o)
 
                 if (graphQLServer)
                 {
+                    string secretToken = null;
+                    if (graphQLSecretTokenPath is { })
+                    {
+                        var buffer = new byte[40];
+                        new SecureRandom().NextBytes(buffer);
+                        secretToken = Convert.ToBase64String(buffer);
+                        await File.WriteAllTextAsync(graphQLSecretTokenPath, secretToken);
+                    }
                     var graphQLNodeServiceProperties = new GraphQLNodeServiceProperties
                     {
                         GraphQLServer = graphQLServer,
                         GraphQLListenHost = graphQLHost,
                         GraphQLListenPort = graphQLPort,
+                        SecretToken = secretToken,
                     };
+                    Console.Error.WriteLine(secretToken);
```

Comment:
Is this output necessary?

Reply:
It was my mistake that remained after debugging and I amended it. Thanks!